sql/system.go

```diff
@@ -61,6 +61,19 @@ CREATE TABLE system.zones (
   id     INT PRIMARY KEY,
   config BYTES
 );`
+
+	// Cluster Reporting table schema
+	reportingTableSchema = `
+CREATE TABLE system.reporting (
+  email       STRING PRIMARY KEY,
+  optin       BOOLEAN,
+  dismissed   INT,
+  firstname   STRING,
+  lastname    STRING,
+  company     STRING,
+  updates     BOOLEAN,
+  lastUpdated TIMESTAMP
+);`
```

Comment:
Is this table ever going to have more than a single row? Breaking out all of the fields like this is classic schema design, but it makes adding new fields more irritating. As an alternative, did you consider having a `system.ui` table which looks like:

```
CREATE TABLE system.ui (
  key STRING PRIMARY KEY,
  json BLOB
)
```

The "reporting" data would be stored under the `"reporting"` key and all of the details would be in the `json` column. New details could be added without adjusting the schema. And then this `ui` table could be used to store other `ui` state as well.

I wouldn't necessarily change anything in this PR, but we should revisit when @cuongdo provides JSON endpoints in #4298.